lib/intercom-rails/script_tag.rb

```diff
@@ -26,6 +27,19 @@ def initialize(options = {})
       self.nonce = options[:nonce]
     end
 
+    def remove_user_cookie_on_logout
+      if (find_current_user_details == {} && controller.response.request.cookies["intercom-session-#{IntercomRails.config.app_id}"])
```

Comment:
Should we be checking for `self.user_details` here? Also, can we extract the cookie name into a variable?